cmd/kubeadm/app/constants/constants.go

```diff
@@ -171,6 +171,11 @@ const (
 
 	// DefaultCIImageRepository points to image registry where CI uploads images from ci-cross build job
 	DefaultCIImageRepository = "gcr.io/kubernetes-ci-images"
+
+	// CoreDNS points to the version of CoreDNS
+	CoreDNS = "CoreDNS"
+	// KubeDNS points to the version of kube-dns
+	KubeDNS = "kube-dns"
```

Comment:
`KubeDNS defines a variable used internally when referring to the kube-dns addon for a cluster`